tests/fixtures/motor/fluid_fixtures.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from rocketpy import Fluid
+
+
+@pytest.fixture
+def nitrous_oxide_fluid():
```

Comment:
maybe referencing in the fixture name this is specific for variable density?

nitrous_oxide_fluid_non_constant_fluid